OZON.ru.js

```diff
@@ -9,596 +9,130 @@
 	"inRepository": true,
 	"translatorType": 4,
 	"browserSupport": "gcsibv",
-	"lastUpdated": "2014-04-03 18:47:00"
+	"lastUpdated": "2015-09-20 18:13:00"
 }
 
-/**
-	OZON.ru translator. Only supports "Book" "Item Type".
-	Copyright (c) 2012 Victor Rybynok, v.rybynok@gmail.com
 
-	Description
-	===========
+function detectWeb(document, url) {
+	var URLRegexp = /https?:\/\/www\.ozon\.ru\/context\/detail\/id\/\d+\//;
+    var isURLValid = URLRegexp.test(url);
```

Comment:
* please pass the translator through http://jsbeautifier.org/ (tab for indent)
* if you're not planning to re-use a particular variable, it's probably better to not create it. e.g. `/https?:\/\/www\.ozon\.ru\/context\/detail\/id\/\d+\//.test(url)`
* variable names should start in lower case (unless they reference a class) and should follow camelCase conventions
* In this case, you want to place the URL filter regexp in the `target` property at the top of the translator. `detectWeb` is only called if the url passes that test.